### ISSUE TITLE
Properties are generated with newlines between each row

### ISSUE THREAD

**Issue by ilvetz** [cloud-atlas-ai/obsidian-am#43]
Notes generated by the plugin have blank lines betweeen each property.

**Comment by muness** [comment]
Can you elaborate or share an example of what you see vs. what you expect? Thanks!

**Comment by ilvetz** [comment]
> Can you elaborate or share an example of what you see vs. what you expect? Thanks!

Of course. Notes in Obsidian include properties (enclosed in three dashes `---`) and the body of the note. When I synchronize the notes each property generated by the plugin is correctly reported on a line but an empty one is added.

![image](https://github.com/cloud-atlas-ai/obsidian-am/assets/5400713/d2ebfa6c-43b4-4ce7-8126-9ee75c0a0af6)
